exe-unit/components/gsb-http-proxy/src/http_to_gsb.rs

```diff
@@ -74,44 +83,52 @@ impl HttpToGsbProxy {
         };
 
         let msg = GsbHttpCallMessage {
-            method,
-            path,
+            method: method.clone(),
+            path: path.clone(),
             body,
             headers: Headers::default().filter(&headers),
         };
 
-        let response = match &self.binding {
-            BindingMode::Local => bus::service(&self.bus_addr).call(msg).await,
-            BindingMode::Net(binding) => {
-                ya_net::from(binding.from)
-                    .to(binding.to)
-                    .service(&self.bus_addr)
-                    .call(msg)
-                    .await
-            }
-        };
+        let endpoint = self.endpoint();
 
-        let result = response.unwrap_or_else(|e| Err(HttpProxyStatusError::from(e)));
+        log::info!("Proxy http {method} `{path}` call to [{}]", endpoint.addr());
+        let result = endpoint
+            .call(msg)
+            .await
+            .unwrap_or_else(|e| Err(HttpProxyStatusError::from(e)));
 
         match result {
-            Ok(r) => HttpToGsbProxyResponse {
-                body: actix_web::web::Bytes::from(r.body.msg_bytes)
-                    .try_into_bytes()
-                    .map_err(|_| {
-                        Error::GsbFailure("Failed to invoke GsbHttpProxy call".to_string())
-                    }),
-                status_code: r.header.status_code,
-                response_headers: r.header.response_headers,
-            },
-            Err(err) => HttpToGsbProxyResponse {
-                body: actix_web::web::Bytes::from(format!("Error: {}", err))
-                    .try_into_bytes()
-                    .map_err(|_| {
-                        Error::GsbFailure("Failed to invoke GsbHttpProxy call".to_string())
-                    }),
-                status_code: StatusCode::INTERNAL_SERVER_ERROR.as_u16(),
-                response_headers: HashMap::new(),
-            },
+            Ok(r) => {
+                log::debug!(
```

Comment:
Is `debug` log level enabled by default?
Do we want to collect these logs by default? On requestor side? On provider side?

Reply:
They are not. I changed to info to simplify, but probably this log should be on debug when we will merge this later to master